semester_course_student/serializers.py

```diff
@@ -0,0 +1,9 @@
+from rest_framework import serializers
+
+from semester_course_student.models import SemesterCourseStudent
+
+
+class SemesterCourseStudentSerializer(serializers.ModelSerializer):
```

Comment:
Burada da son derste yaptığımız Get ve Post için ayrı serializer yazın ve get için foreign key değerlerini de döndüren bir yapı kurun.